docs/UserGuide.md

```diff
@@ -170,101 +359,127 @@ Format: `find a/[LOCATION], [LOCATION], [LOCATION], ...`
+* Partial words will be matched e.g. `diab` will match `Diabetes`.
 * Patients matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Covid Dengue` will return patients with address containing `Covid` or `Dengue`.
+  e.g. `Diabetes, Myopia` will return patients with address containing `Diabetes` or `Myopia`.
 * Conditions must be separated by commas, and whitespaces before and after each condition will be ignored.
 
 Examples:
-* `find con/covid, dengue, ebola`
+* `find con/diabetes, high blood pressure`
+
+<img src="images/FindConditionCommandResult.png" alt="result for 'find serangoon'" width="800"/>
+
 Common mistakes:
-* `find ic/S1234567X` (only condition, name and address can be searched with find)
-  * If you would like to find a person with NRIC, use the [read](#read-specific-patients--read) command instead.
+* `find ic/S1234567X` (only condition, name and address can be searched with find)<br>
+**TIP**: If you would like to find a person with NRIC, use the `read` command instead.
 
-### Deleting a patient : `delete`
+<br>
+
+### Deleting a patient: `delete`
 
 Deletes the specified patient from ImmuniMate.
 
 Format: `delete <NRIC>`
 
-* Deletes the patient with corresponding NRIC.
-* The NRIC must follow the correct format as specified in [Field summary](#field-summary).
+* Deletes the profile of the patient with the corresponding NRIC.
+* The NRIC must follow the correct format as specified in [Field Summary](#field-summary).
 
 Examples:
-* `delete S1234567A` deletes patient uniquely identified by NRIC S1234567A.
+* `delete S1234567A`
+  * deletes patient uniquely identified by NRIC S1234567A.
+
 Common mistakes:
 * `delete S12345678` (NRIC must be in the correct format)
 * `delete` (NRIC must be provided)
 
-### Deleting information of a patient : `deleteinfo`
+<br>
+
+### Deleting information of a patient: `deleteinfo`
 
 Deletes specified optional information from the specified person from ImmuniMate.
 
 Format: `deleteinfo <NRIC> <Field>`
 
 * Deletes specified information of the patient with corresponding NRIC.
-* The NRIC must follow the correct format as specified in [Field summary](#field-summary).
-* The fields must be the optional fields specified in the [Field summary](#field-summary).
 
 Examples:
 * `deleteinfo S1234567A e/`
   * Deletes the email of patient uniquely identified by NRIC S1234567A.
 * `deleteinfo S0123456A e/ bt/ c/`
   * Deletes the email, blood type and country of nationality of patient uniquely identified by NRIC S0123456A.
+
 Common mistakes:
-* `deleteinfo S1234567A abc/` (a valid optional field from the [Field summary](#field-summary) must be provided)
+* `deleteinfo S1234567A abc/` (a valid optional field from the [Field Summary](#field-summary) must be provided)
 * `deleteinfo S1234567A` (a field must be provided)
 * `deleteinfo S1234567A ic/` (mandatory fields cannot be deleted)
+  * If mandatory fields are to be changed, the `update` command should be used.
+
```

Comment:
if the user's intention is to "delete" the field, they should not be table to do so even with "update"